sentry-ruby/lib/sentry/span.rb

```diff
@@ -90,11 +90,11 @@ def initialize(
 
     # Finishes the span by adding a timestamp.
     # @return [self]
```

Comment:
We forgot to mention that it could return `nil` too (early return)

Reply:
I think it should never return nil, will also fix